### PR TITLE
Link GCP project using gcloud cli

### PR DESCRIPTION
**Pull request by azure562**
- Add a new section to use gcloud for project transfer
- Update screenshots

See CMP-4987

### CODE REVIEW

**Review by norosa — CHANGES_REQUESTED** [review]
Instead of blanking out areas of the screen, can we please use fake data of some sort? I typically edit it directly in my browser before taking the screenshot.

I think the banked-out areas are distracting, and conversely, the fake data, if chosen correctly, should give a good indication of what _type_ of values they should be seeing.

**Comment by norosa** [comment]
Asked @yuvaldrori for a review. If he approves, I will merge with the screenshots as is. We can discuss our screenshots style guide separately. Since the UI has changed so much since many of our screenshots were generated, and given that we're dropping the pre-processed drop shadow, we are due for a whole-site refresh soon.

**Review by norosa — APPROVED** [review]
Dismissed my previous review (see previous comment). Here's the required approval from @DocOps. :)

**Comment by norosa** [comment]
Squashed changes and resolved conflicts with `main`. I will mark this PR for auto-merge once we have a second approval from @yuvaldrori.

**Comment by norosa** [comment]
Pushed some fixups. The tests should be good to go now.

**Comment by norosa** [comment]
Preview URL here: https://deploy-preview-160--doitintl-help.netlify.app/

**Comment by norosa** [comment]
The specific changes can be previewed on this page: https://deploy-preview-160--doitintl-help.netlify.app/docs/google-cloud/transfer-google-cloud-projects/transfer-manually

File: website/docs/google-cloud/transfer-google-cloud-projects/transfer-manually.mdx [hunk 1]
```diff
@@ -19,63 +19,102 @@ To follow this tutorial, your Google Cloud account must have the following permi
...
+```
+
+### Link multiple projects
+
+It's also possible to link multiple projects to a billing account using one `gcloud` command.
+
+1. Run the command below to list all the projects for which you have at least one permission.
+
+   ```
+   gcloud projects list --format="value(project_id)" | xargs -I {} echo gcloud beta billing projects link {} --billing-account billing_account
```
Comment (kesor) [inline website/docs/google-cloud/transfer-google-cloud-projects/transfer-manually.mdx:77]: Should be more obvious here that the word `billing_account` at the end of this line is the DoiT provided number, like in line 67 `0X0X0X-0X0X0X-0X0X0X`, but each customer might receive a unique one from DoiT.

File: website/docs/google-cloud/transfer-google-cloud-projects/transfer-manually.mdx [hunk 2]
```diff
@@ -19,63 +19,102 @@ To follow this tutorial, your Google Cloud account must have the following permi
...
+   ```
+
+   - `--format="value(project_id)"`: [Format](https://cloud.google.com/sdk/gcloud/reference#--format)s the result to return the project id.
+   - `|`: Pipes the result of the left side of the pipe (|) into the right side of the pipe.
+   - `xargs`: Builds and executes command lines from standard input.
+   - `-I {}`: Replace occurrences of {} in the initial arguments with names read from standard input.
+   - `echo`: Displays a line of text. Dry run mode.
```
Comment (kesor) [inline website/docs/google-cloud/transfer-google-cloud-projects/transfer-manually.mdx:84]: Maybe note here that `billing_account` piece at the end of the line is the number of the billing account provided by DoiT.

**Review by kesor — APPROVED** [review]
Two small suggestions, one left on the text file, the second suggestion -

The screenshots for Google Console originally had "cmp-playgrounds.com" as the customer's organization. The new screenshots show "doit-intl.com" at places. Not major issue, but would be nice to not show "doit-intl.com" as it might lead to confusion.

Other than these, looks good.

**Comment by norosa** [comment]
I have tidied up the shell code a little to make it more readable, but more importantly, by formatting the commands as I have done, we can use the Docsaurus feature that lets us highlight the (very important) line with the `echo` statement used to determine if the command will be a dry-run or not.

As a bonus, I have added syntax highlighting for shell code.

It seems we have a conflict somewhere in our Docusaurus vs Prism CSS variables, the latter being used for code highlighting. The result is that the highlighted lines were far too dark for our current theme. So, along with my changes, I have added a small CSS root variables fix, using the same color value as used on the main Docusaurus website.

File: website/docs/google-cloud/transfer-google-cloud-projects/transfer-manually.mdx (hunk 1 above)
Comment (norosa) [reply website/docs/google-cloud/transfer-google-cloud-projects/transfer-manually.mdx]: Good shout. Fixed.

File: website/docs/google-cloud/transfer-google-cloud-projects/transfer-manually.mdx (hunk 2 above)
Comment (norosa) [reply website/docs/google-cloud/transfer-google-cloud-projects/transfer-manually.mdx:87]: Fixed!

**Comment by norosa** [comment]
I have completely rewritten the `gcloud` section:

- Rephrased some parts for clarity
- Separated the instructions to replace variables into their own paragraphs
- Added a call out for the highlighted line with the `echo` command
- Reformatted the list of command parts as a definition list
- Pulled some information out into admonitions

**Comment by norosa** [comment]
@kesor has reviewed my rewrite of the `gcloud` section and is happy with it. I have his approval to merge.

**Comment by norosa** [comment]
> The screenshots for Google Console originally had "[cmp-playgrounds](https://doitintl.atlassian.net/browse/CMP-playgrounds).com" as the customer's organization. The new screenshots show "doit-intl.com" at places. Not major issue, but would be nice to not show "doit-intl.com" as it might lead to confusion.

Good point. We are going to do a screenshot refresh soon, and that will solve this issue. Unfortunately, Qing has not been able to get a playground account as of yet.

**Pull request merged**